docs/roles/project-coordinator.rst

```diff
@@ -11,23 +11,23 @@ responsibilities and role of the project coordinator.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
-
+The project coordinator is responsible for ensuring that all club projects have the resources needed to continue developing. This may include monetary resources, connections to mentors or the like and infrastructure to host the projects. 
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-< basic eligiblity requirements – what if someone is on co-op? away from
-campus? working remote? is under academic suspension (i.e. low GPA)?
+The project coordinator must maintian good academic standing as defined by RIT to remain a member of eboard.
+Due to the hands on nature of the role, the project coordinator must be available in person during club and eboard meetings. If needed, a temporary replacement my be appointed to fill the role while the project coordinator is out on coop. In that case there should be good communication between the the two people in order to facilitate a smooth transition.
 
 
 Responsibilities
 ----------------
 
-< responsibilities and routine work that someone in this position is directly
-accountable for – what do they do and what should other eboard members expect
-from this person? >
-
+* Be physically present at all eboard meetings
+* Be present at all RITLUG meetings unless a conflict has come up and eboard has been notified
+* Be available to RITLUG members
+* Act as liason between project leads and the rest of eboard
+* Assist with general eboard tasks as needed
 
 Resources
 ---------
```

Comment:
Could you add some links to this section? This could be some of the GitHub repositories you frequently use as well as certain websites, like tigeros.ritlug.com. It can also include any helpful guides or documentation that was useful for you in building TigerOS.